explain-overview.md

```diff
@@ -6,500 +6,52 @@ aliases: ['/docs/dev/query-execution-plan/','/docs/dev/reference/performance/und
-
-`IndexMerge` allows the optimizer to use multiple indexes per table, and merge the results returned by each index to generate the execution plan of the latter `IndexMerge` in the figure above. Here the `IndexMerge_16` operator has three child nodes, among which `IndexRangeScan_13` and `IndexRangeScan_14` get all the `RowID`s that meet the conditions based on the result of range scan, and then the `TableRowIDScan_15` operator accurately reads all the data that meets the conditions according to these `RowID`s.
-
-For the scan operation that is performed on a specific range of data, such as `IndexRangeScan`/`TableRangeScan`, the `operator info` column in the result has additional information about the scan range compared with other scan operations like `IndexFullScan`/`TableFullScan`. In the above example, the `range:(1,+inf]` in the `IndexRangeScan_13` operator indicates that the operator scans the data from 1 to positive infinity.
-
-> **Note:**
->
-> At present, the `IndexMerge` feature is disabled by default in TiDB 4.0.0-rc.1. In addition, the currently supported scenarios of `IndexMerge` in TiDB 4.0 are limited to the disjunctive normal form (expressions connected by `or`). The conjunctive normal form (expressions connected by `and`) will be supported in later versions. Enable the `IndexMerge` in one of two ways:
->
-> - Set the `tidb_enable_index_merge` system variable to 1;
->
-> - Use the SQL Hint [`USE_INDEX_MERGE`](/optimizer-hints.md#use_index_merget1_name-idx1_name--idx2_name-) in the query.
->
-> SQL Hint has a higher priority than system variables.
-
-### Read the aggregated execution plan
+`EXPLAIN` does not execute the actual query. [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md) can be used to both `EXPLAIN` _and_ execute the query. This can be useful in diagnosing cases where the execution plan selected is suboptimal.
 
-Aggregation algorithms in TiDB include the following categories:
+## Understanding EXPLAIN output
 
-- [Hash Aggregate](#hash-aggregate-example)
-- [Stream Aggregate](#stream-aggregate-example)
+The following describes the output of the `EXPLAIN` statement above:
 
-#### `Hash Aggregate` example
-
-The `Hash Aggregation` operator is optimized in multi-threaded concurrency. It is quick to execute at the cost of more memory usage. The following is an example of `Hash Aggregate`:
-
-{{< copyable "sql" >}}
-
-```sql
-explain select /*+ HASH_AGG() */ count(*) from t;
-```
+* `id` describes the name of an operator, or sub-task that is required to execute the SQL statement. While the structure appears as a tree, executing the query does not strictly require the child nodes to be completed before the parent nodes. TiDB supports intra-query parallelism, so a more accurate way to describe the execution flow is that the child nodes _feed into_ their parent nodes. Parent, child and sibling operators _might_ potentially be executing parts of the query in parallel.
```

Comment:
It could also say "flows into" or "provides data to the parent node" or "delivers data to the parent node".

flows is the most accurate because it implies streaming.

Reply:
I went with "flows into".